app/client/cypress/integration/community/search.spec.js

```diff
@@ -8,5 +8,46 @@ describe('Community search', () => {
     cy.findByPlaceholderText('Search by address', { exact: false }).type(zip);
     cy.findByText('Go').click();
     cy.url().should('include', `/community/${zip}/overview`);
+    cy.findByText('Your Waters: What We Know').should('exist');
+  });
+
+  it('searching for gibberish displays an error', () => {
+    const search = 'jkljl';
+    cy.findByPlaceholderText('Search by address', { exact: false }).type(
+      search,
+    );
+    cy.findByText('Go').click();
+    cy.url().should('equal', `${window.location.origin}/community`);
+    cy.findByText('Data is not available for this location.', {
+      exact: false,
+    }).should('exist');
+  });
+
+  it('searching for a valid huc properly routes to the community overview page', () => {
+    const search = '020700100103';
+    cy.findByPlaceholderText('Search by address', { exact: false }).type(
+      search,
+    );
+    cy.findByText('Go').click();
+    cy.url().should('include', `/community/${search}/overview`);
+    cy.findByText('Your Waters: What We Know').should('exist');
+  });
+
+  it('Clicking the “Use My Location” button, with geolocation enabled, properly geolocates user and routes to the community overview page for the user’s location', () => {
+    cy.mockGeolocation();
+
+    cy.findByText('Use My Location').click();
+
+    cy.url().should('match', /community\/.*\/overview/);
+    cy.findByText('Your Waters: What We Know').should('exist');
```

Comment:
After searching the Cypress docs, I saw you created this custom command (which is great)... I saw you added default params for lat/long, but you're not asserting on the results of the geolocation itself, which you could do (since you're always providing the same lat/long). I'd also suggest passing in the explicit lat/long, just so it's more clear to someone just seeing this test for the first time how a mock method would always return an exact address. I know they could/should look at the function signature but still, I think it's better to communicate more info here if you can.

```suggestion
    const address = 'Scott Cir NW, Washington, District of Columbia, 20036';
    cy.mockGeolocation(false, 38.9072, -77.0369);
    cy.findByText('Use My Location').click();
    cy.url().should('include', `/community/${encodeURI(address)}/overview`);
    cy.findByText('Your Waters: What We Know').should('exist');
```

Reply:
I had originally shied away from checking the location because that address is dependent on the esri geocoder. I figured the scoring system could change and therefore change the address returned and break the test. 

I did some more digging and it looks like if you pass in coordinates you always get one candidate, so maybe it's not really an issue.

Reply:
I went ahead and committed this change.

Reply:
Thanks, @cschwinderg – you make a good point re: being dependent on ESRI's geolocation service not changing their results, but in this case, I'd hope the wouldn't...and if they ever do, we can update the test.